config.ru

```diff
@@ -6,7 +6,7 @@ require 'travis/yml/web'
 env = Travis::Yml::Web::Env
 
 if env.production?
-  use Rack::SslEnforcer
+  use Rack::SslEnforcer unless ENV['TRAVIS_ENTERPRISE']
```

Comment:
we do not use env vars for configuration. There is a library for this, which can take options from file and env vars.

Reply:
https://github.com/travis-ci/travis-yml/blob/master/Gemfile#L13

Reply:
https://github.com/travis-ci/travis-yml/blob/master/lib/travis/yml/web/config.rb#L4